test/views/declaration/ConsigneeDetailsViewSpec.scala

```diff
@@ -305,6 +306,16 @@ class ConsigneeDetailsViewSpec extends UnitViewSpec with CommonMessages with Stu
       }
     }
 
+    onClearance { implicit request =>
+      "display 'Back' button that links to 'Is Exs?' page" in {
+
+        val backButton = createView(navigationForm = ConsignorEoriNumber).getElementById("back-link")
+
+        backButton.text() mustBe messages(backCaption)
+        backButton.attr("href") mustBe routes.IsExsController.displayPage().url
+      }
+    }
+
     onJourney(DeclarationType.SUPPLEMENTARY) { implicit request =>
       "display 'Back' button that links to 'Representative Details' page" in {
```

Comment:
This case should be tested for all journeys. But I can fix it during my rebase if you want.

Reply:
Yeah, if you can it will be great